server/chat-plugins/help.ts

```diff
@@ -3,7 +3,8 @@
  * Uses Regex to match room frequently asked question (RFAQ) entries,
  * and replies if a match is found.
  * Supports configuration.
- * Written by mia-pi.
+ * Written by Mia.
+ * @author mia-pi-git
```

Comment:
There's no need to use your Git username for `@author`, I think you can just use whatever you want.

Reply:
https://jsdoc.app/tags-author.html

Reply:
Yes, i'm just picky about how I have it.